std/algorithm/sorting.d

```diff
@@ -1930,14 +1930,18 @@ unittest
 
 // schwartzSort
 /**
+Returns the same output as an equivalent call to $(D sort), but it will be 
+faster when the sort comparison requires an expensive computation. 
```

Comment:
This sentence is not quite accurate, as `sort` does more than just return something (it mutates the underlying range). Maybe a better wording might be:
````
Sorts a range, just like `sort`, but runs faster than `sort` when the sort comparison requires an expensive computation, by caching computed sort keys beforehand.
````